src/scheduler/gc_work.rs

```diff
@@ -186,6 +188,11 @@ impl<E: ProcessEdgesWork> GCWork<E::VM> for StopMutators<E> {
             mmtk.scheduler.work_buckets[WorkBucketStage::Prepare].add(ScanStackRoot::<E>(mutator));
         });
         trace!("stop_all_mutators end");
+
+        trace!("VM prepare start");
+        <E::VM as VMBinding>::VMCollection::vm_prepare(worker.tls);
```

Comment:
It is just added for symmetry.  We had `vm_release` but not `vm_prepare`.  In `vm_release`, we do the same thing as `RefEnqueue`.

But since it is the same point as `Collection::stop_all_mutators`, I'll remove this and use `Collection::stop_all_mutators` instead.

Reply:
I think it is fine to add `vm_prepare` if it is called in `Prepare`. Otherwise, it is confusing.

Reply:
I removed `vm_prepare` because I only need it to do assertion.  I moved the assertion to `stop_all_mutators`.  If we later find that we need `vm_prepare`, we can add it back.